scout/commands/load/variants.py

```diff
@@ -153,6 +153,8 @@ def variants(
                 gene_obj=gene_obj,
                 build=case_obj["genome_build"],
             )
+            # Update case variants count
+            adapter.case_variants_count(case_obj["_id"], case_obj["owner"], True)
```

Comment:
Same here, category or if None was intended:
```suggestion
            adapter.case_variants_count(case_obj["_id"], case_obj["owner"], update_case=True)
```